src/bun.js/bindings/bindings.cpp

```diff
@@ -2455,6 +2455,17 @@ JSC__JSValue JSC__JSValue__fromEntries(JSC__JSGlobalObject* globalObject, ZigStr
     return JSC::JSValue::encode(object);
 }
 
+JSC__JSValue JSC__JSValue__keys(JSC__JSGlobalObject* globalObject, JSC__JSValue objectValue) {
+    JSC::VM& vm = globalObject->vm();
+
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    JSC::JSObject* object = JSC::JSValue::decode(objectValue).getObject();
```

Comment:
```suggestion
    JSC::JSObject* object = JSC::JSValue::decode(objectValue).toObject(globalObject);
```
`toObject` will coerce to an object, which is needed for strings in this situation